lib/_emerge/EbuildBuild.py

```diff
@@ -343,9 +343,34 @@ def _fetch_exit(self, fetcher):
             and opts.buildpkg != "n"
         )
 
+        # Do not build binary cache for packages from volatile sources.
+        # For volatile sources (eg., git), the PROPERTIES parameter in
+        # the ebuild is set to 'live'.
+
+        # The default behavior is to build binary cache for all pkgs.
+        # "buildpkg-live" is a FEATURE that is enabled by default.
+        # To not build binary cache for live pkgs, we disable it by
+        # specifying FEATURES="-buildpkg-live"
+
+        buildpkg_live = True
+
+        if not "buildpkg-live" in features:
+            buildpkg_live = False
+
+        live_ebuild = False
+
+        if "live" in self.settings.get("PROPERTIES", "").split():
+            live_ebuild = True
+
+        binpkg_live_disabled = (
+            "buildpkg" in features and live_ebuild and not buildpkg_live
```

Comment:
I think it's better to omit `"buildpkg" in features`, since we already have `"buildpkg" in features or self._issyspkg` in the boolean expression that follows, and checking `"buildpkg" in features` in both expressions would be redundant.